### PR TITLE
Support span logs

### PR DESCRIPTION
**Pull request by song-hao**

### CODE REVIEW

File: wavefront_sdk/direct.py [hunk 1]
```diff
@@ -290,3 +298,15 @@ def send_span_now(self, spans):
         @type spans: list[str]
         """
         self._batch_report(spans, self.WAVEFRONT_TRACING_SPAN_FORMAT)
+
+    def send_span_log_now(self, span_logs):
+        """
```
Comment (ustinov) [inline wavefront_sdk/direct.py]: Put the caption onto first line

File: wavefront_sdk/proxy.py [hunk 2]
```diff
@@ -232,3 +237,21 @@ def send_span_now(self, spans):
...
+        Have to construct the data manually by calling
+        common.utils.tracing_span_to_line_data()
+
+        @param span_logs: List of string span log data
+        @type span_logs: list[str]
+        """
+        for span_log in span_logs:
+            try:
+                self._tracing_proxy_connection_handler.send_data(span_log)
+            except Exception as error:
```
Comment (ustinov) [inline wavefront_sdk/proxy.py]: Please use a specific exception name you'd expect here.

File: wavefront_sdk/common/utils.py [hunk 3]
```diff
@@ -261,3 +264,20 @@ def tracing_span_to_line_data(name, start_millis, duration_millis, source,
...
+    @param span_logs: Span Log
+    @param scrambler: Additional UUID, optional
+    @return: Span Log in JSON Format
+    """
+    span_log_json = {'traceId': str(trace_id),
+                     'spanId': str(span_id),
+                     'logs': span_logs}
+    if scrambler:
+        span_log_json['_scrambler'] = str(scrambler)
+    return json.dumps(span_log_json, default=lambda o: o.__dict__) + '\n'
```
Comment (ustinov) [inline wavefront_sdk/common/utils.py]: Just curious, what do we need `default` here for? Do we expect any non-string data to be processed?
The `.__dict__` for an arbitrary object would simply return all it's public attributes/methods

Comment (song-hao) [reply wavefront_sdk/common/utils.py]: We only support string data for span logs

Comment (ustinov) [reply wavefront_sdk/common/utils.py]: It means we don't need to override the default as any object from conversion table [0] is supported.
* [0] https://docs.python.org/3/library/json.html#py-to-json-table

**Comment by vikramraman** [comment]
lgtm. Why are the tests failing?

**Comment by ustinov** [comment]
@song-hao, I was expecting all comments to be addressed before merging this one

**Comment by song-hao** [comment]
> @song-hao, I was expecting all comments to be addressed before merging this one

@ustinov What's left

File: wavefront_sdk/common/utils.py (hunk 3 above)
Comment (song-hao) [reply wavefront_sdk/common/utils.py]: Won't work since SpanLog is a customized class, which is not JSON serializable. Will remain this way.

**Pull request merged**